app/src/forms/form/middleware/auth.js

```diff
@@ -0,0 +1,70 @@
+const keycloak = require('../../../../src/components/keycloak');
```

Comment:
Good to go for now, but hopefully we can not have to reproduce a lot of the stuff in these auth.js files for each silo we add?

Reply:
yup, a lot of that if it turns out to be common can come out and be shared across all the forms.